Gameplay/entities/player/PlayerController.cpp

```diff
@@ -407,7 +457,7 @@ void Hachiko::Scripting::PlayerController::CancelAttack()
 	_attack_current_duration = 0.f;
 	if (_state == PlayerState::RANGED_ATTACKING && _current_bullet >= 0)
 	{
-		BulletController* bullet_controller = _bullet_emitter->GetComponent<BulletController>();
+		CombatManager* bullet_controller = _bullet_emitter->GetComponent<CombatManager>();
```

Comment:
I don't remember quite well, but why we were not making this GetComponent<CombatManager> that are the same always in the awake/start?